sns_ik_lib/include/sns_ik/sns_ik.hpp

```diff
@@ -41,14 +41,14 @@ namespace sns_ik {
   public:
     SNS_IK(const std::string& base_link, const std::string& tip_link,
            const std::string& URDF_param="/robot_description",
-           double looprate=0.005, double eps=1e-5,
+           double loopPeriod=0.01, double eps=1e-5,
            sns_ik::VelocitySolveType type=sns_ik::SNS);
 
     SNS_IK(const KDL::Chain& chain,
            const KDL::JntArray& q_min, const KDL::JntArray& q_max,
            const KDL::JntArray& v_max, const KDL::JntArray& a_max,
            const std::vector<std::string>& jointNames,
-           double looprate=0.005, double eps=1e-5,
+           double loopPeriod=0.01, double eps=1e-5,
```

Comment:
We are using two different types of argument capitalization here: name_underscore and camelCase. We should choose one and make it consistent throughout. Not a super high priority, but it should be done before we advertise this API.